src/app/dashboard/(user-dashboard)/_components/layout/navbar/organisation-switcher.tsx

```diff
@@ -14,21 +15,48 @@ import {
 import { Skeleton } from "~/components/ui/skeleton";
 import { useOrgContext } from "~/contexts/orgContext";
 import { useLocalStorage } from "~/hooks/use-local-storage";
+import { auth } from "~/lib/auth";
 import { CreateOrganization } from "../../create-organization";
 
+const getCurrentOrganization = async (orgId: string) => {
+  const payload = { isActive: true };
+  try {
+    const session = await auth();
+    const response = await axios.put(
+      `https://api-csharp.boilerplate.hng.tech/api/v1/users/organisations/${orgId}`,
```

Comment:
so in the case of other teams what will this do

Reply:
this keeps track of the active account switched to on the organization